### PR TITLE
Microsoft account association issues

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-d4ba295599/how-to/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-de81c69669/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f2e3dc9677/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7589171ea6/how-to/)

**Comment by stephamon** [comment]
Hey @cyrichardson can you squeeze this in today? It's a bit of a rush job :)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-f00a6c21a5/how-to/)

**Comment by cyrichardson** [comment]
On it now, @stephamon.

**Comment by cyrichardson** [comment]
@stephamon Suggested changes here: 
[microsoft-account-association-issues_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2496668/microsoft-account-association-issues_cyrEdits.docx)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-ad90666196/how-to/)

File: content/exchange/microsoft-account-association-issues.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+   You are asked to verify ownership of the new alias. Proceed with sending the address an ownership verification email.
+
+8. After you have successfully changed the primary alias, select **Remove** next to the affected email address to remove it from your Microsoft account.
+
+After your email address is no longer associated with your Microsoft account, restart your device and open Outlook.
+
+### Additional information
+
+Microsoft also provides detailed instructions on completing the task of changing the primary alias for a Microsoft account in [Rename your personal account](https://support.microsoft.com/en-us/help/11545/microsoft-account-rename-your-personal-account).
```
Comment (RackerKate) [inline content/exchange/microsoft-account-association-issues.md]: Suggest "instructions about how to change the primary alias..."

File: content/exchange/microsoft-account-association-issues.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+For more information about how Outlook 2016 implements Autodiscover, you can refer to [Outlook 2016 Implementation of Autodiscover](https://support.microsoft.com/en-us/help/3211279/outlook-2016-implementation-of-autodiscover).
```
Comment (RackerKate) [inline content/exchange/microsoft-account-association-issues.md]: Suggest "Autodiscover, see..."

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-906f87f4b3/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a4b56a78db/how-to/)

**Pull request merged**